test/regress/regress1/nl/iand-native-1.smt2

```diff
@@ -1,5 +1,6 @@
 ; COMMAND-LINE: --iand-mode=value --no-check-models
 ; COMMAND-LINE: --iand-mode=sum --bvand-integer-granularity=1 --finite-model-find --no-check-models
+; COMMAND-LINE:  --solve-bv-as-int=iand --iand-mode=bitwise
```

Comment:
I think we should test other granularities.
My suggestion is to add the following `COMMAND-LINE` to the following test:

https://github.com/CVC4/CVC4/blob/master/test/regress/regress3/bv_to_int_and_or.smt2

```
--solve-bv-as-int=iand --iand-mode=bitwise --bvand-integer-granularity=1
--solve-bv-as-int=iand --iand-mode=bitwise --bvand-integer-granularity=2
--solve-bv-as-int=iand --iand-mode=bitwise --bvand-integer-granularity=4
--solve-bv-as-int=iand --iand-mode=bitwise --bvand-integer-granularity=5
--solve-bv-as-int=iand --iand-mode=bitwise --bvand-integer-granularity=6
```
(the last one is intended to check whether decrementing by 1 is enough in a comment above).

Reply:
Cool, added with b75799ab04315f836e963f352a415b9dbf4f593b